plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenCoordinatesUi.java

```diff
@@ -39,50 +33,30 @@
 
```

Comment:
Looking at this now I'm a little surprised this class doesn't extend Composite or Group.